examples/gno.land/r/demo/banktest/z_0_filetest.gno

```diff
@@ -21,7 +21,6 @@ func main() {
 	println("main before:", mainbal) // plus OrigSend equals 300.
 
 	// simulate a Deposit call.
-	std.TestSetOrigPkgAddr(banktestAddr)
```

Comment:
this is a valid testing case, we should not remove.

Reply:
0dd861efce7035e74befec96b395624539730134